packages/clarity-js/src/data/metadata.ts

```diff
@@ -90,13 +110,7 @@ function supported(target: Window | Document, api: string): boolean {
 }
 
 function track(): void {
-  if (config.track && supported(document, Constant.Cookie)) {
-    let expiry = new Date();
-    expiry.setDate(expiry.getDate() + Setting.Expire);
-    let expires = expiry ? Constant.Expires + expiry.toUTCString() : Constant.Empty;
-    let value = `${data.userId}${Constant.Semicolon}${expires}${Constant.Path}`;
-    document.cookie = Constant.CookieKey + Constant.Equals + value;
-  }
+  setCookie(Constant.CookieKey, data.userId, Setting.Expire);
```

Comment:
Don't we lose the structure of the cookie this way?

Reply:
Sorry not sure if I fully follow. I'm keeping the same format for user cookie as before. Did I miss something here?